### DISCUSSION TITLE
Multiple filter in View Data search field

### DISCUSSION THREAD

**Discussion by shodan02** [arnoudkooi/SN-Utils#331] [General]
Hello there!
I just wanted the use some basic data from the View Data section. But in the search field there is no option for multiple filtering, searching for e.g "name" , "class", ... i'm only able to search or  filter for one item only.

![image](https://user-images.githubusercontent.com/119402002/204473721-9177678e-90a2-4fab-afed-9f91e561b4df.png)

**Comment by arnoudkooi** [comment]
I'll enable regex search. With a pilar `|` you can then do a or search.
Will include that in the next update and let you know when it is published.

![CleanShot 2022-11-29 at 10 29 25@2x](https://user-images.githubusercontent.com/840781/204491680-e7bae1cb-1095-44ff-92ad-83b2ea0d11f0.png)

**Reply by shodan02** [reply]
Perfect , Thank You!!!!

**Reply by arnoudkooi** [reply]
6.3.1.7 is now published, let me know if it works

**Reply by shodan02** [reply]
It is working! Thank You very much!!
